src/components/PokemonContainer/PokemonContainer.tsx

```diff
@@ -113,7 +119,7 @@ const PokemonContainer: React.FC = () => {
         />
         {!currentPokemon &&
           Array.isArray(allPokemons) &&
```

Comment:
I see `allPokemons` is always array
in this case it doesn't make sense to check if it is array
otherwise fix type